<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Downloader</class>
    <widget class="QWidget" name="Downloader">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Downloader</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget#Downloader
                {
                background-color:white
                }
            </string>
        </property>
        <widget class="QLineEdit" name="lineEdit_url">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>512</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="placeholderText">
                <string>请输入下载网址</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_path">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>32</y>
                    <width>512</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="placeholderText">
                <string>请输入下载路径</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_start">
            <property name="geometry">
                <rect>
                    <x>512</x>
                    <y>0</y>
                    <width>64</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_start
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_start:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_start:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>开始</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_choose">
            <property name="geometry">
                <rect>
                    <x>512</x>
                    <y>32</y>
                    <width>64</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_choose
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_choose:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_choose:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>选择</string>
            </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>64</y>
                    <width>1000</width>
                    <height>554</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QListWidget#listWidget
                    {
                    border:none
                    }
                </string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
